arrays/arrays.c

```diff
@@ -21,11 +21,15 @@ typedef struct Array {
  *****/
 Array *create_array (int capacity) {
   // Allocate memory for the Array struct
-
+  Array *array = malloc(sizeof(Array *));
   // Set initial values for capacity and count
-
+  array->capacity = capacity;
+  array->count = 0;
   // Allocate memory for elements
-
+  for (int i = 0; i < capacity; i++) {
+    array->elements[i] = malloc(sizeof(char *));
```

Comment:
This is good, you could also use calloc() here to save yourself the trouble of a for loop

Reply:
oh that's neat